internal/ingress/controller/template/template_test.go

```diff
@@ -448,7 +448,7 @@ func TestBuildAuthResponseHeaders(t *testing.T) {
 		"proxy_set_header 'H-With-Caps-And-Dashes' $authHeader1;",
 	}
 
-	headers := buildAuthResponseHeaders(externalAuthResponseHeaders)
+	headers := buildAuthResponseHeaders(proxySetHeader(nil), externalAuthResponseHeaders)
```

Comment:
@kd7lxl is that possible to add some unit_test for this specific case?

Reply:
just need add a case to `TestProxySetHeader`

Reply:
How's 3f04fbac9 look?

Reply:
LGTM